src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceRentDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+
+  let totalAmount = days * priceRentDay;
+
+  if (days >= longTerm) {
+    totalAmount -= longTermDiscount;
```

Comment:
You can remove the `else` keyword and simply use an `if` statement here, as the `return` statement inside the first `if` block will exit the function if the condition is met.